dependencies/libayatana-appindicator3-0.1.yaml

```diff
@@ -0,0 +1,59 @@
+name: libayatana-appindicator
+buildsystem: cmake-ninja
+config-opts:
+  - -DENABLE_BINDINGS_MONO=NO
+  - -DENABLE_BINDINGS_VALA=NO
+  - -DENABLE_GTKDOC=NO
+sources:
+  - type: git
+    url: https://github.com/AyatanaIndicators/libayatana-appindicator.git
+    tag: 0.5.93
+    commit: 238c8b02718fa5b4af95ede72beeed762094f4cc
+    x-checker-data:
+      type: anitya
+      project-id: 18446
+      tag-template: $version
+modules:
+  - name: ayatana-ido
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/AyatanaIndicators/ayatana-ido.git
+        tag: 0.10.1
+        commit: 13402a2cc4616b4b5f4244413599e635fcfc1401
+        x-checker-data:
+          type: anitya
+          project-id: 18445
+          tag-template: $version
+
+  - name: libayatana-indicator
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/AyatanaIndicators/libayatana-indicator.git
```

Comment:
this is already downloaded on line 9 in a module with the same name. Looks like duplication.

Reply:
two wrongs don't make a right.